src/components/AnnotationsPane.jsx

```diff
@@ -52,7 +53,7 @@ class AnnotationsPane extends React.Component {
         svgPoints.push(
           <circle
             key={svgPointPrefix+i}
-            cx={point.x} cy={point.y} r={10} fill="#c33"
+            cx={point.x} cy={point.y} r={10} fill="#00CED1"
```

Comment:
Now that we're more closely following the colour rules of Issue #64 instead of using placeholder colours, we should (soon) take a look at storing these variables into something like, say, `src/app-colours.js`, or use CSS to define these colours. (Disclaimer: my CSS-fu on SVG elements isn't that fantastic.)

Reply:
That makes sense. I don't think you're allowed to use inline constants with HTML, so moving it into CSS would be best. We also need to figure out a way to notify components that an annotation is selected, as that annotation line will change to a green color.